BookingsApi/BookingsApi.DAL/Migrations/20210804134353_AddUserRoleStaffMember.cs

```diff
@@ -0,0 +1,62 @@
+﻿using BookingsApi.DAL.Helper;
+using BookingsApi.Domain.Enumerations;
+using BookingsApi.Domain.RefData;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddUserRoleStaffMember : Migration
+    {
+        int staffMemberUserId = UserRoleForHearingRole.UserRoleId["Staff Member"];
+
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
+                table: nameof(UserRole),
+                columns: new[] {"Id", "Name"},
+                values: new object[,]
+                {
+                    {staffMemberUserId, "Staff Member"}
+                });
+            
+            var caseRoleId = 213;
+            for (int caseTypeId = 1; caseTypeId <= 38; caseTypeId++)
+            {
+                migrationBuilder.InsertData(
+                    nameof(CaseRole),
+                    new[] { "Id", "Name", "Group", "CaseTypeId" },
+                    new object[,]
+                    {
+                        { caseRoleId, "Staff Member", (int) CaseRoleGroup.StaffMember, caseTypeId }
+                    });
+                caseRoleId++;
+            }
+            
```

Comment:
Direct references to Ids during migrations can lead to foreign key ref identity issues 
Is there a way to implement a dictionary?

Reply:
The Id's are not foreign keys, they are the primary keys of the table.